modules/cluster-logging-feature-reference.adoc

```diff
@@ -0,0 +1,116 @@
+
+
+.Normalizations and Transformations
+[options="header"]
+|============================================================================
+| Feature                                | Description  | Fluentd  | Vector
+| Viaq data model - app                  |              | &#10003; | &#10003;
+| Viaq data model - infra                |              | &#10003; | &#10003;
+| Viaq data model - infra(journal)       |              | &#10003; | &#10003;
+| Viaq data model - audit linux          |              | &#10003; |
+| Viaq data model - audit kube api       |              | &#10003; | &#10003;
+| Viaq data model - audit openshift api  |              | &#10003; | &#10003;
+| Viaq data model - ovn                  |              | &#10003; | &#10003;
+| Loglevel Normalization                 |              | &#10003; | &#10003;
+| JSON parsing                           |              | &#10003; |
+| Structured Index                       |              | &#10003; |
+| Multiline error detection              |              | &#10003; |
+| Multicontainer / Split indices         |              | &#10003; |
+| Flatten labels                         |              | &#10003; | &#10003;
+| CLF static labels                      |              | &#10003; | &#10003;
+|============================================================================
+
+.Security and Compliance
+[options="header"]
+|=====================================================
+| Feature           | Description  | Fluentd  | Vector
+| FIPS              |              |          |
+| Export compliant  |              |          |
+|=====================================================
+
+.Tuning
+[options="header"]
+|==========================================================
+| Feature                | Description  | Fluentd  | Vector
+| Fluentd readlinelimit  |              | &#10003; |
+| Fluentd buffer         |              | &#10003; |
+| - chunklimitsize       |              | &#10003; |
+| - totallimitsize       |              | &#10003; |
+| - overflowaction       |              | &#10003; |
+| - flushthreadcount     |              | &#10003; |
+| - flushmode            |              | &#10003; |
+| - flushinterval        |              | &#10003; |
+| - retrywait            |              | &#10003; |
+| - retrytype            |              | &#10003; |
+| - retrymaxinterval     |              | &#10003; |
+| - retrytimeout         |              | &#10003; |
+|==========================================================
+
+.Metrics
+[options="header"]
+|=====================================================
+| Feature         | Description  | Fluentd  | Vector
+| logs collected  |              | &#10003; |
+| logs produced   |              | &#10003; | &#10003;
+| dash board      |              | &#10003; | &#10003;
+| Alerts          |              | &#10003; |
+|=====================================================
+
+.Miscellaneous
+[options="header"]
```

Comment:
Missing the mention for disconnected clusters.

Reply:
Could you clarify please?

Reply:
One of the optional post-installation tasks for OCP is to convert the cluster to a disconnected one ([docs](https://docs.openshift.com/container-platform/4.10/post_installation_configuration/connected-to-disconnected.html)). This is like IPv6 another flavor of OCP that we need to have explicitly on the support matrix.

Reply:
Added.